Cargo.toml

```diff
@@ -40,7 +40,7 @@ cfg-if = { version = "1.0" }
 contract-build = { version = "4.0.0-rc.3" }
 darling = { version = "0.20.8" }
 derive_more = { version = "0.99.17", default-features = false }
-drink = { version = "=0.11.0" }
+drink = { version = "=0.11.0", default-features = false, features = ["std"] }
```

Comment:
This is causing the build to fail because the `session` feature is missing https://github.com/ascjones/drink/blob/14330cfd8925b1b8fb633a0620cf586aeb67e54c/drink/Cargo.toml#L41